resources/views/content/styles.twig

```diff
@@ -1,2 +1,2 @@
-{#<link rel="stylesheet" href="{{ plugin_path("Findologic") }}/css/findologic.min.css" />#}
-<link rel="stylesheet" href="https://localhost:8080/findologic_ceres.css" />
+<link rel="stylesheet" href="{{ plugin_path("Findologic") }}/js/dist/findologic_ceres.css" />
+{#<link rel="stylesheet" href="https://localhost:8080/findologic_ceres.css" />#}
```

Comment:
Please add TODO here as well

Reply:
As discussed I've added a section to the `README.md`